src/prometheus_alert_rules/heartbeat.rule

```diff
@@ -0,0 +1,7 @@
+# Based on https://awesome-prometheus-alerts.grep.to/rules.html#prometheus-self-monitoring-1
```

Comment:
```suggestion
# Based on https://awesome-prometheus-alerts.grep.to/rules.html#prometheus-self-monitoring-1
# This is an always-firing alert for indicating if alertmanager itself is down (e.g. cos-alerter).
```

Reply:
I believe we have this in the summary already.